roles/setup_tftp/tasks/main.yml

```diff
@@ -0,0 +1,23 @@
+---
+- name: Setup TFTP
+  block:
+    - name: Install TFTP
+      ansible.builtin.package:
+        name: tftp-server
+        state: present
+
+
+    - name: Start TFTP
+      ansible.builtin.service:
+        name: tftp.socket 
+        state: started
+        enabled: true
+
+    - name: Allow incoming tftp traffic
+      ansible.posix.firewalld:
+        zone: public
+        service: tftp
+        permanent: yes
+        state: enabled
+        immediate: yes
+  become: true
```

Comment:
Please move the `become` to the top of the block:
```yaml
- name: Setup TFTP
  become: true
  block:
    - name: Install TFTP
      # ...
```

Reply:
+1